concent_api/core/constants.py

```diff
@@ -10,6 +10,9 @@
 # Defines max length of task_id passed in Golem Messages.
 MESSAGE_TASK_ID_MAX_LENGTH = 36
 
+#  Defines regex for database migrations to valid uuid
+REGEX_FOR_VALID_UUID = r'^[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}\Z'
```

Comment:
is there a unit test for this reg-ex?

Reply:
After your comment added: https://github.com/golemfactory/concent/pull/765/files#diff-c78970958ce2a71cb3883f3ee77b630dR8